prac_07/project_managment.py

```diff
@@ -0,0 +1,57 @@
+from prac_07.project import Project
+
+MENU_STRING = "(L)oad projects\n(S)ave project\n(D)isplay projects\n(F)ilter projects by date\n(A)dd new peojects\n(U)pdate projects\n(Q)uit"
+
+
+def main():
+    print(MENU_STRING)
+    choice = input(">>> ").upper()
+    while choice != "Q":
+        if choice == "L":
+            load_project()
+        elif choice == "S":
+            print("S")
+        elif choice == "D":
+            display_project(projects)
+        elif choice == "F":
+            print("filter")
+        elif choice == "A":
+            print("add")
+        elif choice == "U":
+            print("update")
+        else:
+            print("Invalid menu choice")
+            print(MENU_STRING)
+        choice = input(">>> ").upper
+
+
+def load_project():
+    in_file = open("projects.txt", "r")
+    projects = []
+    for line in in_file:
+        line = line.strip()
+        parts = line.split(',')
+        projects.append(parts)
```

Comment:
You were on the right track with splitting the line up into parts. Those parts are not projects though. You would need to create a new Project to append to projects.

I suggest trying to get at least this function working for your learning. Being able to do this will help you in the future as this is a very common type of programming task (creating a list of objects).